README.md

```diff
@@ -48,10 +68,13 @@ When using `permissionsMiddleware()` you can pass a custom message object with a
 By default it is 
 ```js
 {
-  user = (ctx) => `You're the following permissions: ${ctx.command.userPerms.join(', ')}`,
-  my = (ctx) => `I'm missing the following permissions: ${ctx.command.myPerms.join(', ')}`
+  my = (ctx) => `I am missing the following permissions: ${ctx.command.myPerms.filter(p => !ctx.myPerms(p)).map(p => (typeof humanReadable === 'function' ? humanReadable(ctx, p) : humanReadable[p]) || p).join(', ')}`,
```

Comment:
It would probably be better if these `my` and `user` passed the missing permissions as if the user has to do the check for missing permissions, there is no point to use this. Additionally, passing through this specific function as an option creates issues with the `humanReadable`. Essentially, the paramaters `humanReadable` and `my` and `user` would actually be exclusive. I suggest changing `msgFunction` to
```js
type msgFunction = (perms: (keyof typeof bits)[],  humanReadable: humanReadableBits, ctx: CommandContext) => string
```
or something similar.

Reply:
can u elaborate

Reply:
Currently, if I want to change the function to return `Bonk! I'm missing the permissions: Kick Members, Ban Members`, with the current changes I would need to copy the `humanReadable` object. There is no way to access it in the passed function. Ex. 
```js
worker.commands.middleware(permissionsMiddleware({
  my: (ctx) => `I am missing the following permissions: ${ctx.command.myPerms.filter(p => !ctx.myPerms(p)).map(p => (typeof humanReadable === 'function' ? humanReadable(ctx, p) : humanReadable[p]) ?? p).join(', ')}`
}))
```
Throws the error: `humanReadable is not defined`. This is because the `humanReadable` would need to be in the scope of the defined function. 
By passing both the missing permissions and the `humanReadable` object, I could make it
```js
worker.commands.middleware(permissionsMiddleware({
  my: (perms, humanReadable, ctx) => `Bonk! I'm missing the permissions: ${perms.map(p => humanReadable[p] ?? p).join(', ')}`
}))
```
Which is much more readable and reduces duplicated code.

Reply:
permissionMiddleware.humanReadable is accessible tho.

Reply:
but yes i see where you are coming from

Reply:
I think passing `humanReadable` to the function is unnecessary and limiting, so no. I was gonna suggest that you could make it optionally `perms` to be the readable, however I don't even like that because it's unexpected.

Reply:
Oh, you were exporting it, I didn't think about that.

Reply:
I think it would be more unexpected to be given a missing permissions function without the missing permissions. At the very least, it would make it easier and more readable being able to just do `perms.join(', ')` rather than filtering the command's permissions.

Reply:
so no changes?

Reply:
well it would still be `perms.map(p => (typeof humanReadable === 'function' ? humanReadable(ctx, p) : humanReadable[p]) ?? p).join(', ')`

Reply:
unless you are thinking about putting the map to human readable outside the my and user. then there would be no need for passing it into the function

Reply:
Sure, just don't pass humanReadable to the function that looks ugly.

Reply:
Only for code that must account for both. A user would only be able to pass an object OR a function. It would therefore be
```js
perms.map(p => humanReadable(ctx, p) ?? p).join(', ')
```
OR
```js
perms.map(p => humanReadable[p] ?? p).join(', ')
```
A user would not need to include both

Reply:
okay so just perms and ctx

perms will be pre human readablified before going into my and user

Reply:
No they shouldn't be, special ops can be used to recognize the missing permissions so they shouldn't be made human before being passed